packages/lit-dev-content/site/learn/learn.11tydata.js

```diff
@@ -87,11 +87,23 @@ const loadArticleData = async () => {
 const loadVideoData = () =>
   [
     {
-      title: 'Chat with Lit #1 – Westbrook Johnson (Adobe) ',
+      title: 'Declarative Reactive Web Components with Justin Fagnani',
+      summary: `Justin Fagnani's talk covering the fundamental ideas of lit-html and LitElement. This talk introduces why Lit is architected the way it is.`,
```

Comment:
His full name is already in the title. I added the point about web components since the beginning of the video covers the promise of web components and why Lit exists.
```suggestion
      summary: `Justin covers what Web Components are and how LitElement and lit-html add value on top of the native APIs. This talk covers the fundamentals of how and why Lit is architected the way it is.`,
```